admin/users.php

```diff
@@ -189,18 +189,18 @@ function init() {
                   <td class="password">&nbsp;</td>
                   <td class="confirm">&nbsp;</td>
                 <?php } ?>
-                <?php if ($action == 'edit' || $action == 'password') { ?>
-                  <?php if ($user == $userDetails['id']) { ?>
+                <?php if ($action == 'add' || $action == 'edit' || $action == 'password') { ?>
+                  <?php if ($user == $userDetails['id'] && $action != 'add') { //steve for equal td on edit?>
                     <td class="actions">
-                      <button type="submit" class="btn btn-primary"><?php echo IMAGE_UPDATE; ?></button> <a href="<?php echo zen_href_link(FILENAME_USERS) ?>" class="btn btn-default" role="button"><?php echo IMAGE_CANCEL; ?></a>
+                            <button type="submit" class="btn btn-primary"><?php echo IMAGE_UPDATE; ?></button><a href="<?php echo zen_href_link(FILENAME_USERS) ?>" class="btn btn-default" role="button"><?php echo IMAGE_CANCEL; ?></a>
                     </td>
                   <?php } else { ?>
                     <td class="actions">&nbsp;</td>
                   <?php } ?>
                 <?php } elseif ($action != 'add') { ?>
                   <td class="actions">
                       <?php if ($action != 'delete') { ?>
-                      <a href="<?php echo zen_href_link(FILENAME_USERS, 'action=edit&user=' . $userDetails['id']) ?>" class="btn btn-primary" role="button"><?php echo IMAGE_EDIT; ?></a> <a href="<?php echo zen_href_link(FILENAME_USERS, 'action=password&user=' . $userDetails['id']) ?>" class="btn btn-primary"><?php echo IMAGE_RESET_PWD; ?></a>
+                            <a href="<?php echo zen_href_link(FILENAME_USERS, 'action=edit&user=' . $userDetails['id']) ?>" class="btn btn-primary" role="button"><?php echo IMAGE_EDIT; ?></a> <a href="<?php echo zen_href_link(FILENAME_USERS, 'action=password&user=' . $userDetails['id']) ?>" class="btn btn-primary"><?php echo IMAGE_RESET_PWD; ?></a>

```

Comment:
I'm confused by this double-length indentation. And above on `195`

Reply:
dunno, wasn't me, phpstorm did it then ran off